### PR TITLE
EZP-31114: Added "skip-indexing" option for install command

### PR DESCRIPTION
**Pull request by ilukac**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-31114](https://jira.ez.no/browse/EZP-31114)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | `7.5`/`8.x` 
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

Added `--disable-index` option for `ezplatform:install` command. 

**TODO**:
- [x] Ask for Code Review.

### CODE REVIEW

File: eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php [hunk 1]
```diff
@@ -61,6 +62,12 @@ protected function configure()
             InputArgument::REQUIRED,
             'The type of install. Available options: ' . implode(', ', array_keys($this->installers))
         );
+        $this->addOption(
+            'disable-index',
```
Comment (gggeek) [inline eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php]: skip-indexation ?

Comment (ilukac) [reply eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php]: Yeah, naming is always difficult :)

skip-indexing sounds good enough for me, wdyt?

Comment (adamwojs) [reply eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php]: @ilukac`skip-indexing` is fine

**Comment by adamwojs** [comment]
Could you also please create JIRA issue? I will be easier for us to track the current state of your PR (QA, doc, create release notes etc.)

**Comment by ilukac** [comment]
@adamwojs name changed, jira task added

**Comment by adamwojs** [comment]
> @adamwojs name changed, jira task added

Thank you @ilukac

**Comment by lserwatka** [comment]
@alongosz could you merge it up?

**Comment by alongosz** [comment]
Merged up.

Thank you @ilukac :tada:

**Pull request merged**